internals/overlord/state/identities.go

```diff
@@ -266,17 +324,43 @@ func (s *State) Identities() map[string]*Identity {
 
 // IdentityFromInputs returns an identity with the given inputs, or nil
 // if there is none.
-func (s *State) IdentityFromInputs(userID *uint32) *Identity {
+//
+// Identity priority:
+//  1. If both username and password are provided, the function attempts to
+//     match a basic type identity. The userID is ignored in this case. If
```

Comment:
Nit: just to clarify it's a value and not just a normal word, let's add quotes:
```suggestion
//     match a "basic" type identity. The userID is ignored in this case. If
```